test/e2e/install/install_test.go

```diff
@@ -288,7 +288,7 @@ var _ = Describe("[Install] durability", func() {
 			By("checking there are no leftovers")
 			// by taking the ns from the ds we're avoiding the need to figure out in advanced
 			// at which ns we should look for the resources
-			mf, err := rte.GetManifests(configuration.Plat, configuration.PlatVersion, ds.Namespace, true)
+			mf, err := rte.GetManifests(configuration.Plat, configuration.PlatVersion, ds.Namespace, true, true)
```

Comment:
time to cleanup this signature if time allows

Reply:
Will do, but I wouldn't block this PR for that.

Reply:
of course no, this must to be a separate independent effort